src/software/pipeline/main_LdrToHdrMerge.cpp

```diff
@@ -150,6 +150,9 @@ int aliceVision_main(int argc, char** argv)
 
     const std::size_t channelQuantization = std::pow(2, channelQuantizationPower);
 
+    // Fusion always produces linear image. sRGB is the only non linear color space that must be changed to linear (sRGB linear). 
+    image::EImageColorSpace mergedColorSpace = (workingColorSpace == image::EImageColorSpace::SRGB) ? image::EImageColorSpace::LINEAR : workingColorSpace;
```

Comment:
I don't understand this. You're loading in workingcolorspace, which may be srgb, but after, you do not convert from srgb because of this.